tests/webservice_action_step_test.php

```diff
@@ -108,7 +108,7 @@ public function test_with_non_existent_function() {
         $step = new \tool_trigger\steps\actions\webservice_action_step(json_encode($stepsettings));
 
         // Manually catch the exception to check the message
-        $this->expectException('dml_missing_record_exception');
+        $this->expectException('Error');
```

Comment:
Would it be better to just tear out this check, and only check the exception message?

Reply:
I did try this, but PHP unit doesn't like it if you try and doing anything with an exception, without first calling `expectException`